content/en/docs/languages/rust/libraries.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Using instrumentation libraries
+linkTitle: Libraries
+weight: 40
+description: How to instrument libraries an app depends on
```

Comment:
```suggestion
description: How to use instrumentation libraries
```

Reply:
maybe stick with what we already have in other pages 
https://github.com/open-telemetry/opentelemetry.io/blob/main/content/en/docs/languages/net/libraries.md?plain=1#L2-L4

Reply:
I picked that description because people might not know what an instrumentation library is, bay having the title "Using instrumentation libraries" this would feel redundant

Reply:
nvm, its inconsistent across languages...https://github.com/open-telemetry/opentelemetry.io/blob/main/content/en/docs/languages/js/libraries.md?plain=1#L5

Reply:
Eventually we should have descriptions for all the pages, but this is yet another consistency challenge.

If you have strong objections against this description, I can remove it, otherwise we can keep it and refine it later